### ISSUE TITLE
Replacing Stylus with PostCSS

### ISSUE THREAD

**Issue by her** [iansinnott/react-static-boilerplate#14]
I think deprecating Stylus in favor of PostCSS would be a great enhancement.

**Comment by iansinnott** [comment]
Yeah, I agree. I still use stylus but it's clear that I'm no longer the only person using this repository so I'd like to make it a bit less opinionated and a bit more accessible to anyone who doesn't love stylus. 

Do you have any thoughts on how to best configure loaders? It seems like the best approach would be to let users customize their pre-processing (loaders) as much as they want, but I don't want to build _another_ configuration syntax on top of webpack, so I'm still thinking through what a good approach would be here.

**Comment by iansinnott** [comment]
The latest build has postcss in the build pipeline. This doesn't affect stylus at all but for anyone who wants to add to the postcss pipeline it might be a bit easier now.
